Chapter 3

gregmacfarlane opened the pull request:
This is the transcription pull request for chapter 3.

gregmacfarlane committed:
Add demonstration text

This is text showing the basics of Rmarkdown / bookdown. The text should be deleted when the real chapter 1 gets put in.

We also included a reference to Train / McFadden 1978

gregmacfarlane committed:
Ignore _bookdown compiled outputs

This just contains the images that the R session creates from the code, and can be ignored.

gregmacfarlane committed:
Ignore bookdown-generated RDS file

Not sure where this came from, but doesn't need to be considered.

barnesmaxmax committed:
Add Chapter 2 text

Added text for Chapter 2. No figures or equations are included in the original text, so this chapter is finished.

loganbennett93 committed:
Add text from Chapter 1

gregmacfarlane committed:
Reflow, add paragraphs

gregmacfarlane commented:
- [ ] @bwang0601 has the assignment to do the first copy, and identify tasks for others.

Commit:
add bib for chapter 1-bruce

bwang0601 commented:
- [ ] @loganbennett93 add equations 3.1 to 3.7 add terms definition for the equations as 

- [ ] @barnesmaxmax  add equations 3.8 to 3.14 add terms definition for the equations as well

- [ ] @camlunt add equations 3.15 to 3.21 add terms definition for the equations as well

- [ ] @SamKLau add figure 3.1, add or match equation letters in the text.

Commit:
add chapter 3 text-bruce

SamKLau committed:
Ch.2 references. moved from main branch (oops) to right branch

camlunt committed:
Incorporated references/hyperlinks to citations and chapters

gregmacfarlane committed:
Minor edits to reference formatting

The [@ref; @ref] format helps pandoc understand that these are parenthetical citations.

gregmacfarlane committed:
Update discussion on dataset and examples.

gregmacfarlane committed:
Build Chapter 1 and 2

gregmacfarlane committed:
Merge branch 'transcribe1-2' into transcribe3

camlunt committed:
Added equations 3.15 through 3.21

Includes equation lines, where statements, and in-line variable explanations. Does not include equation numbering or proper indentation/alignment.

loganbennett93 committed:
Add equations 3.1 through 3.7 to chapter 3

There is a symbol in equation 3.1 akin to rho that I could not figure out how to add, and it is currently represented with a "P". I was also unable to figure out how to format the indentations of the equations as in the original book, and have organized the format as I could.

camlunt committed:
Added equation numbering

For equations 3.15 through 3.21.  Tabs/Indentation still not correct.

camlunt committed:
Merge remote-tracking branch 'origin/transcribe3' into transcribe3

gregmacfarlane committed:
Revise math around equations 3.1

gregmacfarlane committed:
Remember that tabs are not spaces!

camlunt committed:
Changed Where statements to markdown lists

for equations 3.15 through 3.21, Where statements are now listed as bullet points.

loganbennett93 committed:
Update equations to markdown list format

I also added some of the in-text variables that were previously missing from the paragraphs.

barnesmaxmax committed:
Add Eq 3.8 to 3.14

Added equations 3.8 to 3.14 add term definitions for equations as well.

loganbennett93 committed:
Fix an error in equation 3.4

loganbennett93 committed:
Update equations to match preferred format

barnesmaxmax committed:
Auto stash before merge of "transcribe3" and "origin/transcribe3"

barnesmaxmax committed:
Merge branch 'transcribe3' into HEAD

barnesmaxmax committed:
Merge cleanup

Cleaning up a merge.

SamKLau committed:
added figure of illustration of deterministic choice

barnesmaxmax committed:
Merge branch 'maxsmess' into transcribe3

SamKLau committed:
added figure of illustration for deterministic choice

SamKLau committed:
adding in mathematical terms in paper

SamKLau committed:
pulled changes for equations 3.1 - 3.7 and merged conflict points

Merge branch 'transcribe3' of github.com:byu-transpolab/modechoice into transcribe3

# Conflicts:
#	03-utility-theory.Rmd

SamKLau committed:
added math terms in text

gregmacfarlane committed:
Build site

gregmacfarlane commented:
Great job!
